src/frontend/src/storage/index.ts

```diff
@@ -2,6 +2,7 @@
 
 import { parseUserNumber } from "$src/utils/userNumber";
 import { nonNullish } from "@dfinity/utils";
+import * as idb from "idb-keyval";
```

Comment:
can you be specific in the import to avoid bundling what's not used?

Reply:
Ah good point!

Reply:
Actually I'll do this separately, everywhere in the codebase.

Reply:
So you'll do in this PR right?

Reply:
Done!